packages/cli/src/commands/deploy/__tests__/baremetal.test.js

```diff
@@ -131,6 +131,7 @@ describe('serverConfigWithDefaults', () => {
 
   it('overrides defaults with custom', () => {
     const serverConfig = {
+      port: 12345,
```

Comment:
I'd love to see two tests that go along with this: one that checks that `serverConfig.port` defaults to 22, and another that verifies that setting it here to 12345 results in `serverConfig.port` also being 12345.

Reply:
I've added a test to check the default value.

I believe your second test condition is already covered by the test you highlight? I think the "overrides defaults with custom" tests that all the properties are properly updated, including the new port value?

Reply:
Yep you're right!